go/vt/vtgate/vstream_manager.go

```diff
@@ -646,6 +665,7 @@ func (vs *vstream) streamFromTablet(ctx context.Context, sgtid *binlogdatapb.Sha
 
 			sendevents := make([]*binlogdatapb.VEvent, 0, len(events))
 			for i, event := range events {
+				vs.vsm.vstreamsEventsStreamed.Add(labels, 1)
```

Comment:
Not every event gets streamed, and we haven't actually streamed it yet here. I think that we should probably do this in `sendAll`, using the length of the slice of events that we sent (after we've streamed them):
```
		select {
		case <-ctx.Done():
			return nil
		case vs.eventCh <- events:
			vs.vsm.vstreamsEventsStreamed.Add(labels, len(events))
		}
```

Reply:
Updated!